crates/uv/src/commands/project/run.rs

```diff
@@ -1016,7 +1016,15 @@ pub(crate) async fn run(
             .as_ref()
             .map(PythonEnvironment::scripts)
             .into_iter()
+            .chain(
+                ephemeral_env
+                    .as_ref()
+                    .and_then(|env| env.python_executable().parent())
+                    .into_iter(),
```

Comment:
In the ephemeral environment case we should never need to do this, right?

Reply:
Yeah good call. That will always be virtual.